sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -1090,6 +1090,18 @@ object SQLConf {
       .booleanConf
       .createWithDefault(true)
 
+  val CACHE_DISABLE_CONFIGS =
+    buildConf("spark.sql.cache.disableConfigs")
+      .doc("Configurations needs to be turned off, to avoid regression for cached query, so that " +
+        "the outputPartitioning of the underlying cached query plan can be leveraged later.")
```

Comment:
I think the comment is for developers, so it is difficult for a user to understand this description. Could you brush up it more?